tilecloud_chain/__init__.py

```diff
@@ -971,8 +1000,7 @@ def __init__(self):
         self.nb = 0
 
     def __call__(self, tile=None):
-        if tile and tile.data:
-            self.nb += 1
```

Comment:
This is reverting a change that broke the metatile counter (it counted before data was populated